pkg/action/cl001/ac000/plan.go

```diff
@@ -36,17 +36,17 @@ var Plan = []plan.Step{
 	},
 	{
 		Action:  "ac004",
-		Backoff: plan.NewBackoff(24*time.Minute, 3*time.Minute),
+		Backoff: plan.NewBackoff(30*time.Minute, 3*time.Minute),
 		Comment: "check worker count",
 	},
 	{
 		Action:  "ac006",
-		Backoff: plan.NewBackoff(10*time.Second, 2*time.Second),
+		Backoff: plan.NewBackoff(15*time.Minute, 1*time.Minute),
```

Comment:
At this point all masters and workers should be there and we wait for all apps to properly run. Why does that take 15 minutes?

Reply:
Yeah so the problem is that we have some pods which are not being deployed as fast as expected, like node-exporter. I'm not sure if this is due to the fact of waiting for tiller/chart-operator but it takes a while.

Reply:
I understand. What would be nice would be to document our reasoning and insights somehow in the code. Maybe in the plan or in the action explainers themselves.

Reply:
Good point, I added some explanation in the action